site/source/docs/api_reference/html5.h.rst

```diff
@@ -2405,3 +2405,9 @@ Functions
 .. c:function:: void emscripten_throw_string(const char *utf8String)
 
   Invokes JavaScript throw statement and throws a string.
+
+.. c:function:: void emscripten_unwind_stack(void)
+
+  Throws a JavaScript exception that unwinds the stack and yields execution
+  back to the browser event loop. This function does not return execution
```

Comment:
`This function does not return execution` - maybe that sentence could be `This halts execution, so code after this call will not be reached.` ?